### PR TITLE
WorkItem(TDKN-213):Daikon exception is not an OSGI plugin

### PR DESCRIPTION
**Pull request by ttao-talendbj**
https://jira.talendforge.org/browse/TDKN-213

**What is the problem this Pull Request is trying to solve?**
 
**What is the chosen solution to this problem?**
 
**Link to the JIRA issue**

 
**Please check if the Pull Request fulfills these requirements**
- [ ] The PR commit message follows our [guidelines](../CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features, coverage should be over 75% in the new code)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in JIRA), if any, are all linked or available in the Pull Request

 
**[ ] This Pull Request introduces a breaking change**

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/107/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/107/)

File: daikon-exception/pom.xml [hunk 1]
```diff
@@ -70,5 +75,55 @@
...
+							<goal>bundle</goal>
+						</goals>
+						<configuration>
+							<classifier>bundle</classifier>
+							<instructions>
+								<Bundle-SymbolicName>org.talend.daikon.exception</Bundle-SymbolicName>
+								<DynamicImport-Package>*</DynamicImport-Package>
+								<Service-Component>*</Service-Component>
+								<Embed-Transitive>true</Embed-Transitive>
+								<Embed-Dependency>*;artifactId=paranamer|snappy-java|commons-compress|xz;inline=true
```
Comment (sgandon) [inline daikon-exception/pom.xml]: I believe those depedencies are not required

File: daikon-exception/pom.xml [hunk 2]
```diff
@@ -70,5 +75,55 @@
...
+								</Embed-Dependency>
+								<Import-Package>
+									com.cedarsoftware.*;resolution:=optional,com.fasterxml.*;resolution:=optional,org.apache.*;resolution:=optional,*
```
Comment (sgandon) [inline daikon-exception/pom.xml]: Also can you make sure that all import packages are really necessary.
I am pretty sure some of them are not necessary.

**Comment by ttao-talendbj** [comment]
Updated.

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/111/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/111/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/112/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/112/)

**Pull request merged**